src/odemis/gui/comp/settings.py

```diff
@@ -19,11 +19,11 @@
     Odemis. If not, see http://www.gnu.org/licenses/.
 
 """
-from builtins import str
-from past.builtins import basestring
 import wx
 from decorator import decorator
 
+import logging
```

Comment:
Is this import necessary?

Reply:
No. I only used it temporarily to test my changes. Good catch!